src/world/climate.c

```diff
@@ -26,6 +26,9 @@
 #include "../util/util.h"
 #include "climate.h"
 
+int SnowLevel;
+int NextSnowLevel;
```

Comment:
We always prefix global variables with a small g, usually then followed by the related module. So in this case they should be called:
```
gClimateSnowLevel
gClimateNextSnowLevel
```